app/src/main/java/projekt/substratum/common/Packages.java

```diff
@@ -424,81 +426,104 @@ public static String getThemeAPIs(Context context,
-                        } else if (target == 26) {
-                            targetSdk = context.getString(R.string.api_26);
-                        } else if (target == 27) {
-                            targetSdk = context.getString(R.string.api_27);
+                        switch (target) {
+                            case Build.VERSION_CODES.M:
+                                targetSdk = context.getString(R.string.api_23);
+                                break;
+                            case Build.VERSION_CODES.N:
+                                targetSdk = context.getString(R.string.api_24);
+                                break;
+                            case Build.VERSION_CODES.N_MR1:
+                                targetSdk = context.getString(R.string.api_25);
+                                break;
+                            case Build.VERSION_CODES.O:
+                                targetSdk = context.getString(R.string.api_26);
+                                break;
+                            case Build.VERSION_CODES.O_MR1:
+                                targetSdk = context.getString(R.string.api_27);
+                                break;
                         }
                         return minSdk + " - " + targetSdk;
                     }
                 } catch (NoSuchFieldError noSuchFieldError) {
                     // The device is API 23 if it throws a NoSuchFieldError
-                    if (appInfo.targetSdkVersion == 23) {
+                    if (appInfo.targetSdkVersion == Build.VERSION_CODES.M) {
                         return context.getString(R.string.api_23);
                     } else {
                         String targetAPI = "";
                         int target = appInfo.targetSdkVersion;
-                        if (target == 24) {
-                            targetAPI = context.getString(R.string.api_24);
-                        } else if (target == 25) {
-                            targetAPI = context.getString(R.string.api_25);
-                        } else if (target == 26) {
-                            targetAPI = context.getString(R.string.api_26);
+                        switch (target) {
+                            case Build.VERSION_CODES.N:
+                                targetAPI = context.getString(R.string.api_24);
+                                break;
+                            case Build.VERSION_CODES.N_MR1:
+                                targetAPI = context.getString(R.string.api_25);
+                                break;
+                            case Build.VERSION_CODES.O:
+                                targetAPI = context.getString(R.string.api_26);
+                                break;
                         }
                         return context.getString(R.string.api_23) + " - " + targetAPI;
                     }
                 }
             }
-        } catch (Exception e) {
+        } catch (Exception unused) {
```

Comment:
`s/unused/ignored/`

Reply:
Can you explain a bit? I'm not sure what you mean

Reply:
Just change `unused` to `ignored` because we prefer it for our code base

Reply:
Ok, thanks